web/components/supabase-search.tsx

```diff
@@ -567,6 +582,7 @@ const useContractSearch = (
       p: isPrizeMarketString,
       fy: forYou,
       mt: marketTier,
+      tf: topicFilter,
```

Comment:
You're adding tf as a parameter, but wouldn't you need to modify the backend to accept it?

Also, there already exists the `topicSlug` parameter. It would be good to reuse that if possible

Reply:
No, see the code further below (line 615)
```
   const newContracts = await searchContracts({
        term: query,
        filter,
        sort,
        contractType,
        offset: freshQuery ? 0 : state.contracts?.length ?? 0,
        limit: CONTRACTS_PER_SEARCH_PAGE,
        topicSlug:
          topicSlug !== ''
            ? topicSlug
            : topicFilter !== ''
            ? topicFilter
            : undefined,
        creatorId: additionalFilter?.creatorId,
        isPolitics: additionalFilter?.isPolitics,
        isPrizeMarket: isPrizeMarketString,
        marketTier,
        forYou,
      })
      ```
      
      I didn't wanna use topicSlug because i didn't want to mess with the topic page usecase too much

Reply:
Ah, ok, I guess this is fine!